### PR TITLE
travis: enable gometalinter

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: goclean.sh [hunk 1]
```diff
@@ -6,9 +6,30 @@
 # 4. race detector (http://blog.golang.org/race-detector)
 # 5. test coverage (http://blog.golang.org/cover)
 
+# gometaling (github.com/alecthomas/gometalinter) is used to run each each
```
Comment (jrick) [inline goclean.sh]: typo

**Comment by dajohi** [comment]
Updated

File: blockmanager.go [hunk 2]
```diff
@@ -659,33 +659,22 @@ func (b *blockManager) startSync(peers *list.List) {
...
-				if !sp.Connected() {
-					return
-				}
-				if b.IsCurrent() {
-					msg := wire.NewMsgGetMiningState()
-					sp.QueueMessage(msg, nil)
-					return
-				}
-			}
+		time.Sleep(3 * time.Second)
```
Comment (jrick) [inline blockmanager.go]: this is completely different behavior, you must keep the loop.  do this: `for range <-ticker.C {`

Comment (jrick) [inline blockmanager.go]: or:

```
for {
    time.Sleep(3 * time.Second)
    // rest of code
}
```
}

Comment (dajohi) [inline blockmanager.go]: this one

**Pull request merged**